packages/@webex/internal-plugin-metrics/src/call-diagnostic/call-diagnostic-metrics-batcher.ts

```diff
@@ -39,6 +39,9 @@ const CallDiagnosticEventsBatcher = Batcher.extend({
     // check event names and append latencies?
     const eventName = item.eventPayload?.event?.name as MetricEventNames;
     const joinTimes: ClientEvent['payload']['joinTimes'] = {};
+    const audioSetupDelay: MediaQualityEvent['payload']['audioSetupDelay'] = {};
+    const videoSetupDelay: MediaQualityEvent['payload']['videoSetupDelay'] = {};
```

Comment:
since you import `MediaQualityEvent` from the `metrics.types` I recommend to create separated types instead refer to a sub-type here for example:

```ts
type MediaQualityEventAudioPayload = MediaQualityEvent['payload']['audioSetupDelay'];
```
then  
```ts
    const audioSetupDelay: MediaQualityEventAudioPayload = {};
```

Reply:
added